.gitignore

```diff
@@ -132,3 +132,4 @@ packages/service/test/tmp
 packages/cli/fixtures/sqlite/db
 packages/client-cli/test/tmp
 packages/db/test/fixtures/schema.lock
+.ccls-cache
```

Comment:
This should arguably go in your personal `.gitignore` but 🤷 it's fine.

Reply:
I didn't know that existed. removed.